lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -22,6 +20,9 @@ Function Write-DebugLog {
 
     Write-Debug $msg
 
+    $log_path = $null
+    $log_path = Get-AnsibleParam -obj $params -name "log_path"
```

Comment:
Please don't make changes to module behaviour in this PR.

Reply:
To add to this, prefer not to make this change, especially for an untested module. We can look at this in a separate PR if you wish.